volume-encryption.md - rewording/grammar (#870)
longhorn/website: content/docs/1.6.0/advanced-resources/security/volume-encryption.md

To create an encrypted volume, you must create a PVC using a StorageClass that has been configured for encryption. The above StorageClass examples can be used as a starting point.

> After creation of the PVC it will remain in the `Pending` state until the associated secret has been created and can be retrieved by the csi `external-provisioner` sidecar. Afterwards, the regular volume creation process continues with encryption taking effect.

Reviewer: ```suggestion
After a PVC is created, it remains in the `Pending` state until the associated secret has been created and can be retrieved by the CSI sidecar `external-provisioner`. Afterwards, the regular volume creation process continues with encryption taking effect.
```
